src/main/java/com/epam/eco/commons/avro/converter/DefaultAvroConverters.java

```diff
@@ -339,16 +349,25 @@ public static class DefaultMapAvroConverter implements AvroConverter {
         public Object toAvro(Object value, Schema schema, AvroConverters converters) {
             if (value instanceof Map) {
                 Schema mapValuesSchema = schema.getValueType();
-                return ((Map<String, Object>) value).entrySet().stream()
-                        .collect(Collectors.toMap(
-                                Map.Entry::getKey,
-                                e -> converters.getForSchema(mapValuesSchema).
-                                    toAvro(e.getValue(), mapValuesSchema, converters)));
+                
+                Map<String, Object> convertedMap = new HashMap<>();
+                
+                Consumer<Map.Entry<String, Object>> enrichMap = (entry) -> {
+                    convertedMap.put(
+                        entry.getKey(),
+                        converters.getForSchema(mapValuesSchema).
+                            toAvro(entry.getValue(), mapValuesSchema, converters)
+                    );
+                };
+                
+                ((Map<String, Object>) value).entrySet()
+                    .forEach(enrichMap);
```

Comment:
Consider using Map.forEach, I believe it will make code more concise <br>

```suggestion
                ((Map<String, Object>) value).forEach(
                        (k, v) -> convertedMap.put(
                                k,
                                converters.getForSchema(mapValuesSchema).toAvro(v, mapValuesSchema, converters)
                        )
                );
```